examples/receive-email-using-client/receive_email_using_client.md

```diff
@@ -1,6 +1,6 @@
 # Receive emails using a client
 
-The email client is used to receive (with POP3 or IMAP4) emails using the SSL or STARTTLS protocols. This sample includes receiving emails with default configurations over SSL using the default ports.
+The email client is used to receive (with POP3 or IMAP4) emails using the SSL or STARTTLS protocols. This sample includes receiving emails with default configurations over SSL using the default ports via POP3. To use IMAP4 refer to [`IMAP client`](https://lib.ballerina.io/ballerina/email/latest/clients/ImapClient).
```

Comment:
Shall we say the usage of IMPA4 is the same as POP3. Simply replacing the POP3 client works right.

Reply:
Updated the wording with [0a28163391659977ec67b93ea74c154737221d24](https://github.com/ballerina-platform/ballerina-distribution/pull/3444/commits/0a28163391659977ec67b93ea74c154737221d24)